### PR TITLE
minor enhancements

### PR DESCRIPTION
**Pull request by duncdrum**
found some oddities while looking into #800. This makes error handling for faulty test-suites more robust, and adds an alternative means of using a local copy when developing. 

There is a regression from upstream with `npm.Install` that still needs more investigation. 

see #649

### CODE REVIEW

File: generators/app/templates/specs/xqs/xqSuite.js [hunk 1]
```diff
@@ -22,31 +24,40 @@ http.get(url, (res) => {
 
   // called when the complete response is received.
   res.on('end', () => {
-    let xqsReport = JSON.parse(data)
-    let xqsPkg = xqsReport.testsuite.package
-    let xqstCount = xqsReport.testsuite.tests
-    let xqstCase = xqsReport.testsuite.testcase
+    // NOTE(DP): XQST errors on testsuite, will be returned as application/xml
```
Comment (joewiz) [inline generators/app/templates/specs/xqs/xqSuite.js:27]: Typo: XQST -> XQTS

**Comment by joewiz** [comment]
@duncdrum Could you explain the purpose of the "exception fixture"? Looking elsewhere in the commit, I don't see where it's used.

**Comment by joewiz** [comment]
@duncdrum A minor thing, not about this PR per se: I just noticed that view.xql wasn't included in the cleanup of XQuery file extensions in https://github.com/eXist-db/generator-exist/pull/701. You consider slipping that into this PR too - as another minor enhancement?

**Comment by duncdrum** [comment]
@joewiz sure the fixtures are normally used in the unit test of the custom mocha interface to exist's `xqsuite`.  they are all based on real responses and corner cases observed in the wild. 

https://github.com/eXist-db/generator-exist/blob/c4585f61d5c358c38517817b1d6b490638fcae48/test/mock_xqs.js#L6-L9

I added the xml in the hope to have an idea for how to use it in a future test, that doesn't involve debugging/fixing exists, mochas, and my own code. My current approach can't handle mocha encountering errors of this type, much like xqsuite can't handle NPEs.

**Review by joewiz — APPROVED** [review]
Thanks for the info on fixtures! That's a neat capability. The PR looks good to me.
